sklearn/decomposition/nmf.py

```diff
@@ -842,8 +842,13 @@ class NMF(BaseEstimator, TransformerMixin):
     solver : 'pg' | 'cd'
         Numerical solver to use:
         'pg' is a (deprecated) Projected Gradient solver.
+
+        .. versionchanged:: 0.17
```

Comment:
@amueller Could you clarify me about this or similar ones 

"Improved speed of ```newton-cg``` solver in :class:`linear_model.LogisticRegression`" 

Does this fall in versionchanged ?